app/engine.py

```diff
@@ -1,163 +1,320 @@
+import csv
+import json
+import logging
+import re
+from pathlib import Path
+
+import joblib
 import pandas as pd
+import psutil
+from nltk.stem.snowball import DutchStemmer
+from settings import LOG_LEVEL
+from sklearn.feature_extraction.text import CountVectorizer, TfidfTransformer
+from sklearn.linear_model import LogisticRegression
 from sklearn.metrics import ConfusionMatrixDisplay
 from sklearn.model_selection import GridSearchCV, train_test_split
 from sklearn.pipeline import Pipeline
-from sklearn.feature_extraction.text import TfidfTransformer, CountVectorizer
-from sklearn.linear_model import LogisticRegression
-from nltk.stem.snowball import DutchStemmer
-import joblib
+
 import nltk
-import re
-import csv
-import psutil
+
+handler = logging.StreamHandler()
+handler.setFormatter(logging.Formatter('%(message)s'))
+
+logger = logging.getLogger(__name__)
+logger.setLevel(level=LOG_LEVEL)
+logger.addHandler(handler)
+
 
 class TextClassifier:
     _text = 'Text'
     _main = 'Main'
     _middle = 'Middle'
     _sub = 'Sub'
     _lbl = 'Label'
-    model = None
 
-    def __init__(self, *args, **kwargs):
-        load_from_disk = kwargs.get('model_from_disk')
-        self._init_lookup()
-        if load_from_disk:
-            self._init_model(load_from_disk)
+    def __init__(self, model_from_disk=None):
```

Comment:
`model_from_disk` is never used, maybe we can just remove it?

Reply:
Removed